ldm/dream/server.py

```diff
@@ -151,6 +151,7 @@ def image_progress(sample, step):
                 {'event': 'step', 'step': step + 1, 'url': path}
             ) + '\n',"utf-8"))
 
+        config['initimg'] = config.pop('initimg_name','')
```

Comment:
Nit: probably makes sense to put this up on [line 89](https://github.com/lstein/stable-diffusion/blob/72df7809429cbf3bba2a0cbc9c2a975b84ae6f15/ldm/dream/server.py#L89) instead?

Reply:
Thanks. I fixed that.